lib/polymorpheus.rb

```diff
@@ -25,6 +25,7 @@ module ConnectionAdapters
   end
 
   def self.require_foreigner?
+    ActiveRecord::VERSION::MAJOR < 5 &&
     !(::ActiveRecord::VERSION::MAJOR >= 4 &&
       ::ActiveRecord::VERSION::MINOR >= 2)
```

Comment:
Not unless we drop Rails 3 support. I don't have a particularly compelling reason to keep supporting this many versions, but the exercise of thinking about _how_ to build a gem that retains backwards compatibility has been interesting.